dashboards-observability/common/constants/application_analytics.ts

```diff
@@ -14,6 +14,23 @@ export const TAB_TRACE_TITLE = 'Traces & Spans';
 export const TAB_LOG_TITLE = 'Log Events';
 export const TAB_CONFIG_TITLE = 'Configuration';
 
+export const APP_ANALYTICS_API_PREFIX = '/api/observability/application'
+
 export interface optionType {
   label: string;
 }
+
+export type ApplicationListType = {
+  name: string;
+  id: string;
+  dateCreated: string;
+  dateModified: string;
+};
+
+export type ApplicationType = {
+  name: string;
+  description: string;
+  query: string;
+  selectedServices: Array<string>;
+  selectedTraces: Array<string>;
+}
```

Comment:
We should move the types [here](https://github.com/opensearch-project/observability/tree/main/dashboards-observability/common/types)

Reply:
Got it!